packages/components/src/components/ResizableBox/ResizableBox.tsx

```diff
@@ -177,6 +180,7 @@ export const ResizableBox = ({
     height,
     minHeight = 0,
     maxHeight,
+    maxSizeDirections,
```

Comment:
I suggest to rename it to something more clear:

e.g.:
```
updateWidthOnWindowResize?: boolean;
updateHeightOnWindowResize?: boolean;
```

Reply:
I am not sure.. Do we really need 2 props if this behavior could sufficiently be handled only by one? It is about the same thing, just the direction is different. And the ResizableBox already has a lot of props..

Reply:
The thing is that when I read `maxSizeDirections ` I don't understand what it does. 

I couldn't figure out one prop for this two use cases without loosing context. And I think having many props is not an issue if you know what they do.

Reply:
Right, makes sense. Haven't figured out better namings so I used your suggestions. Thanks a lot, now the code looks better, more understandable.